translation-concordance.md

```diff
@@ -3,9 +3,9 @@ layout: blank
 title: Translation Concordance
 ---
 
-An automatically-generated list of page translation relationships across this site.
+An automatically-generated list of lessons translation relationships across our publications.
 
-{% assign original_pages = site.pages | where_exp: "item", "item.name != 'redirect.html'" | where_exp: "item", "item.name != 'redirects.json'" | where_exp: "item", "item.name != 'index.md'" | where_exp: "item", "item.original == nil" %}
```

Comment:
Thanks for starting to take this on!

First off, we have to keep all the complex existing filters because that's what prevents the repeat / disjointed table you're currently seeing. In particular the `where_exp: "item", "item.original == nil"` makes sure that each row in the table is fixed to one "original" page (no matter what language the original is).

It would be useful for the concordance to show TWO tables - one for lessons, and one for all other pages. So I'd start by duplicating all the **original** code (before you made these modifications) from `{% assign original_pages...` all the way through the end `</table>` tag.

Then, we'll need to modify both of the `assign` liquid calls. For the one that should only include lessons, I'd add a filter at the end that does `where_exp: "item", "item.layout == 'lesson'"`. For the one that should show all non-lesson pages, `where_exp: "item", "item.layout != 'lesson'"`

Try to get that working first, then we'll double-check how to filter out retired lessons - although I think we might actually want to keep retired lessons in this list since they're still valid pages on our site, just not ones that we list up front in the lessons directory.

Sorry, that's a lot for just one comment - let me know if it makes sense or if I can clarify anything in order for you to try and make the changes I suggested.

Reply:
Thank you, @mdlincoln! I didn't know (as you can guess) that I needed to keep the where_exp code. If we think retired lessons should start, we keep them. 

I am short on Internet connection so I'll probably take a few more days to try to make these changes.